lib/install/management/store_files.go

```diff
@@ -173,3 +173,17 @@ func (d *Dispatcher) getVCHRootDir(vchVM *vm.VirtualMachine) (string, error) {
 	}
 	return path.Join(parent, uuid), nil
 }
+
+func (d *Dispatcher) createVolumeStores(conf *metadata.VirtualContainerHostConfigSpec) error {
+	for _, url := range conf.VolumeLocations {
+		ds, err := d.session.Finder.Datastore(d.ctx, url.Host)
+		if err != nil {
+			return errors.Errorf("Could not retrieve datastore with host %s due to error %s", url.Host, err)
+		}
+		_, err = vsphere.NewDatastore(d.ctx, d.session, ds, url.Path)
```

Comment:
We want the result of this persisted.  Specifically, the `RootURL` in this struct.